docs/user-guide/browser-profiles.md

```diff
@@ -0,0 +1,20 @@
+# Browser Profiles
+
+Browser Profiles are saved instances of a web browsing session that can be re-used to crawl websites as they were configued, with any cookies or saved login sessions. They are specifically useful for crawling websites as a logged in user or accepting cookie consent popups.
+
+!!! warning "Make specific accounts for crawling sites with browser profiles!"
```

Comment:
IMO this warning deserves its own section, like "Before Getting Started" or "Keeping your Accounts Safe"?

Reply:
Personally I like the attention the yellow warning callout provides?  I also like that the warning text is an instruction. I feel like a dedicated section might be easier to skip over as people want to get to the content they came there for?

Also (semi-related) this should be included in the in-app text as well as part of #384.

Reply:
I think separate section makes sense, as there's more to say on this, so added a 'Best practice: Create web archiving-specific accounts'

Reply:
Made it into a collapsible info block to address all the points: still at the top so easy to see, and provides details if user expands. Talks about PII a bit as well while mentioning actual passwords aren't archived.